### ISSUE TITLE
Payment Options

### ISSUE THREAD

**Issue by BigBlockStudios** [castlamp/zenbership#164]
Is there a "Will call" payment option?  CAsh/Cheque COD ....  lots of clubs offer that type of payment.  Kinda important.

**Comment by slove05** [comment]
Not that I have found. It would be nice to have the Invoice be an option when members register for an event. You may be able to create a custom php hook at the beginning of the Event Registration process to create an invoice

**Comment by BigBlockStudios** [comment]
Yes - this feature is critical for me,   I am a director for a fish and hunt club (read as 'gun club') and MOST of our events, memberships, raffle tickets etc are cash in hand.  but things like training and safety courses would be great for online registration and show up with cash on the day. 

I kind of assumed this would be something I had to write,  have not had time to really dig into it, but the payment processor files all look like plugins basically.   Soooo it looks like I am writing that. 

I have not gone as far as registration confirmations/invoicing...  does the system not send a confirmation/invoice when a user registers??  that seems odd? (or is the functionality just broken?)

**Comment by BigBlockStudios** [comment]
just looked at the extension docs ....  ha.  non-existent looks like we are code diving then.

**Comment by slove05** [comment]
If you get anywhere with this let me know. Or if I do I'll let you know.

**Comment by slove05** [comment]
@BigBlockStudios I'm currently researching https://membershipworks.com/ as it has an invoice/will call function built into the event registration.

**Comment by BigBlockStudios** [comment]
@slove05  - did you get any indication of cost?  ours is a very poor non-profit :(

**Comment by slove05** [comment]
https://membershipworks.com/pricing/
Ours is as well. I liked that we could use the existing site. Pit fall is paypal. You have to upgrade to a pro version for $30 and subscriptions makes it $40 but our clients were happy to switch to Stripe as it has lower international fee's and no added cost for subscriptions. We can invoice clients and they get an invoice to bring with payment.
